notebook.py

```diff
@@ -0,0 +1,60 @@
+from collections import UserDict
+from note import Note
+from datetime import datetime
+import os
+import pickle
+
+
+class Notebook(UserDict):
+    def __init__(self):
+        super().__init__()
+        self.filename = 'notes'
+        if os.path.exists(self.filename):
+            self.load_records()
+            self.__id = max([id for id in self])+1
+        else:
+            self.data = {}
+            self.__id = 1
+
+    def load_records(self):
+        if (os.path.getsize(self.filename) > 0):
+            with open(self.filename, 'rb') as f:
+                self.data = pickle.load(f)
+        else:
+            self.data = {}
+
+    def save_records(self):
+        with open(self.filename, 'wb') as f:
+            pickle.dump(self.data, f)
+
+    def create_note(self, text: str):
+        id = self.__id
+        note = Note(id, text, datetime.now())
+        self[id] = note
```

Comment:
self.data[id]?

Reply:
self[id] and self.data[id] , they are same :)